src/materials/02-molecules/headers/lead-header--short.html

```diff
@@ -0,0 +1,16 @@
+<header class="lead-header bg-dust short has-breadcrumbs {{default classes ''}}">
+  <div class="row">
+    <div class="large-12 columns">
+      {{> navigation.breadcrumbs }}
+      <hgroup class="lead-header_group">
+        <h1 class="lead-header_title small c-{{default color 'oil'}}">{{default title 'Page Header'}}</h1>
+        <div class="medium-8 columns no-padding">
+          <p class="lead-header_subtitle caps c-{{default color 'oil'}}">{{default description "21 Clarence Place, San Francisco CA"}}</p>
+        </div>
+        <div class="medium-4 columns no-padding">
+          <span class="lead-header_secondary-action">{{> buttons.link classes="text-only" }}</span>
```

Comment:
What's your thinking in using the button class here, rather than another class for styling the link?